src/shell/CMakeLists.txt

```diff
@@ -177,11 +177,10 @@ FOREACH(T ${LEANTESTS})
   endif()
 ENDFOREACH(T)
 
-# Create a lake test for each subdirectory of `lake/examples` which contains a `test.sh` file.
-file(GLOB_RECURSE LEANLAKETESTS "${LEAN_SOURCE_DIR}/lake/examples/test.sh")
+# Create a lake test for each subdirectory of `lake` which contains a `test.sh` file.
+file(GLOB_RECURSE LEANLAKETESTS "${LEAN_SOURCE_DIR}/lake/test.sh")
 FOREACH(T ${LEANLAKETESTS})
-  if(NOT T MATCHES ".*lake-packages.*")
-  if(NOT T MATCHES ".*bootstrap.*")
+  if(NOT T MATCHES ".*(lake-packages|bootstrap|104|git).*")
```

Comment:
The latter two test cases still reference the old Lake repo root, should I update them?

Reply:
I'll leave that to you then :)